v0/destinations/pardot/data/PardotIdentify.json

```diff
@@ -0,0 +1,277 @@
+      "traits.password",
+      "context.traits.password"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "company",
+        "sourceKeys": [
+      "traits.company",
+      "context.traits.company"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "prospect_account_id",
+        "sourceKeys": [
+            "prospect_account_id",
+            "traits.prospect_account_id",
+            "traits.prospectAccountId",
+            "context.traits.prospect_account_id",
+            "context.traits.prospectAccountId"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "website",
+        "sourceKeys": [
+            "traits.website",
+            "context.traits.website"   
+        ],
+        "required": false
+      },
+      {
+        "destKey": "notes",
+        "sourceKeys": [
+            "traits.notes",
+            "context.traits.notes"   
+        ],
+        "required": false
+      },
+      {
+        "destKey": "job_title",
+        "sourceKeys": [
+            "traits.job_title",
+            "traits.jobTitle",
+            "context.traits.job_title",
+            "context.traits.jobTitle"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "department",
+        "sourceKeys": [
+            "traits.department",
+            "context.traits.department"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "country",
+        "sourceKeys": [
+            "traits.country",
+            "context.traits.country"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "address_one",
+        "sourceKeys": [
+            "traits.address_one",
+            "context.traits.address_one"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "address_two",
+        "sourceKeys": [
+            "traits.address_two",
+            "context.traits.address_two"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "city",
+        "sourceKeys": [
+            "traits.city",
+            "context.traits.city"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "state",
+        "sourceKeys": [
+            "traits.state",
+            "context.traits.state"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "zip",
+        "sourceKeys": [
+            "traits.zip",
+            "context.traits.zip"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "phone",
+        "sourceKeys": [
+            "traits.phone",
+            "context.traits.phone"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "fax",
+        "sourceKeys": [
+            "traits.fax",
+            "context.traits.fax"
+        ],
+        "required": false
+      },
+      {
+        "destKey": "source",
+        "sourceKeys": [
+            "traits.source",
```

Comment:
I think this is for the source from where the user has come. If yes, then we should use `utm_source`

Reply:
should we remove it then?